src/HttpHandler.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Project MyHttpServer
+ * @author Pei-Ru Wang, Cheryl Huang, Yuan Zhou
+ * @version 0.0.1
+ */
+
+
+#ifndef _HTTPHANDLER_H
+#define _HTTPHANDLER_H
+
+#include <boost\asio.hpp>
+#include <boost\shared_ptr.hpp>
+#include <boost\bind.hpp>
+#include <boost\lexical_cast.hpp>
+#include <boost\enable_shared_from_this.hpp>
+
+#include <iostream>
+#include <unordered_map>
+#include <string>
+
+#include "HttpPacket.h"
+
+#define	Asio boost::asio
```

Comment:
Please do not #define types in header.
You can add "using" in the class.

Reply:
done